apps/frontend/vite.config.mts

```diff
@@ -0,0 +1,86 @@
+/// <reference types="vitest" />
+import { resolve } from 'path'
+import { defineConfig, normalizePath } from 'vite'
+
+import react from '@vitejs/plugin-react'
+// viteStaticCopy contains some `require`, so we need to have our config as .mts instead of .ts.
+// https://vitejs.dev/guide/troubleshooting.html#this-package-is-esm-only
+import { viteStaticCopy } from 'vite-plugin-static-copy'
+import viteTsConfigPaths from 'vite-tsconfig-paths'
+
+export default defineConfig({
+  base: '',
+  cacheDir: '../../node_modules/.vite/frontend',
+
+  server: {
+    port: 4200,
+    host: 'localhost',
+    // Allows workers to fetch the needed .ts file
+    fs: {
+      allow: ['../../'],
+    },
+  },
+
+  preview: {
```

Comment:
What is this `preview` used for?

Reply:
https://vitejs.dev/guide/static-deploy.html#testing-the-app-locally

Preview uses the build in the `dist` folder and serves that locally. so you can test prod build locally